src/components/common/Table/Table.js

```diff
@@ -77,11 +77,11 @@ const TableComponent = ({
   };
 
   const selectRole = role => {
-    return role === 'Administrator'
+    return role === 1
```

Comment:
Would love to consider a look into the `===`. What if somehow the spec or API gets changed to send out a String type of 1, 2 or 3? This would be looking at the type AND the value in this case. Might be worth revisiting or noting for future reference. 

I'm not asking for a refactor just a consideration around the implementation.

Reply:
Now that I understand what I'm actually at, this isn't an issue.